app/main/db/queries.py

```diff
@@ -323,6 +323,12 @@ def _build_consignment_view_query(
                 else_=None,
             ),
         ).label("closure_period"),
+        Consignment.ConsignmentId.label("consignment_id"),
+        Consignment.ConsignmentReference.label("consignment_reference"),
```

Comment:
do we need all of these fields?

Reply:
yes to support bread crumb on consignment view, keep the consistent approach to columns across all queries

Reply:
I thought this query was intended for the rows in the table. I think its fine for now but might be worth us revisting all these queries if we have time in a couple sprints time

Reply:
the query is intended get file details, but you do need body, series and consignment ref details for breadcrumb in consignment view, due to that these fields added

Reply:
Yeah that's understandable but it seems a bit hacky. It's fine for now but could be worth revisiting later if we have time

Reply:
I agree, we have to revisit all queries during refactoring the code